### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token '<'

### DISCUSSION THREAD

**Discussion by tunztunztunz** [jeffreylanters/react-unity-webgl#323] [Help Wanted!]
Unity Version: 2020.3.26f1

I've followed all of the issue threads and tried a few different folder structures that I thought would fix the issue, but I can't seem to find what's causing this. 

I'm declaring the context like this
```js
import Unity, { UnityContext } from "react-unity-webgl";

const unityContext = new UnityContext({
  loaderUrl: "../../../../public/build/Build/build.loader.js",
  dataUrl: "../../../../public/build/Build/build.data",
  frameworkUrl: "../../../../public/build/Build/build.framework",
  codeUrl: "../../../../public/build/Build/build.wasm",
});
```

and I've tried

```js
import Unity, { UnityContext } from "react-unity-webgl";

const unityContext = new UnityContext({
  loaderUrl: "../../../../public/build/build.loader.js",
  dataUrl: "../../../../public/build/build.data",
  frameworkUrl: "../../../../public/build/build.framework.js",
  codeUrl: "../../../../public/build/build.wasm",
});
```

and calling the player like this:
```js
<Unity unityContext={unityContext} style={{
        height: "100%",
        width: 950,
        border: "2px solid black",
        background: "grey",
      }}/>
```

I see the unity player styles, but nothing loads. I've tried the following file structures (changing paths to match), but they both give me the same results.

![Screen Shot 2022-02-08 at 12 17 33 PM](https://user-images.githubusercontent.com/53169493/153068645-db81922c-126a-4fd7-abb3-278057552fef.png)

![Screen Shot 2022-02-08 at 12 19 31 PM](https://user-images.githubusercontent.com/53169493/153068925-d09ae422-218c-4cea-9e92-477b9bfe2258.png)

![Screen Shot 2022-02-08 at 12 20 38 PM](https://user-images.githubusercontent.com/53169493/153069090-e56d8594-1ecc-4c77-bafb-14e0b790f71c.png)

I noticed that a lot of the documentation references a .json file that Unity no longer serves in their builds for more recent versions. Is there a different set up for 2020/2021 for this package to work?

**Comment by jeffreylanters** [comment]
Duplicate of #57, also explained in the [documentation](https://github.com/jeffreylanters/react-unity-webgl#understanding-the-unity-context-object). 

TLDR; 

```Diff
- loaderUrl: "../../../../public/build/Build/build.loader.js",
+ loaderUrl: "build/Build/build.loader.js",
```

**Reply by tunztunztunz** [reply]
```js
const unityContext = new UnityContext({
  loaderUrl: "build/Build/build.loader.js",
  dataUrl: "build/Build/build.data",
  frameworkUrl: "build/Build/build.framework.js",
  codeUrl: "build/Build/build.wasm",
});
```
The above doesn't work and I receive the same error even though the paths are now relative to the index.html.

I've noticed two things after re-reading the documentation and the issue:
- #57 also involves a JSON file that I do not have. How is that being handled and do I need to do anything differently for 2020/2021? 
- I also noticed all of these examples don't use the index.html file that comes with the unity build unless they're using that as their index. In my case, I can't do that as the index.html file is part of my application. If this is the case, do I need to add anything there? Could two index.html files be causing this issue?

**Reply by makeanewgame** [reply]
Have you tried getting the build without compression? You need to add uncompressed files while working in the React development environment.
